### DISCUSSION TITLE
[Feature] only new photos

### DISCUSSION THREAD

**Discussion by abbbubble** [immich-app/immich#14722] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

If the mobile device has already backed up photos using another app, I only want to back up new photos now. However, starting the backup process currently causes duplicate backups of the same photos.

So, I suggest:
Including an option to back up only new photos, avoiding duplicate backups.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4169
